### ISSUE TITLE
cobblerd fails to start with no module named items

### ISSUE THREAD

**Issue by cyb-r** [cobbler/cobbler#2147]
Using Ubuntu 18.04 and version cobbler-3.0.1 release tarball I am getting the following messages via `/var/log/cobbler/cobbler.log`

````
Thu Sep 12 15:38:06 2019 - INFO | Exception raised when loading module managers.import_signatures
Thu Sep 12 15:38:06 2019 - INFO | Exception occured: <class 'ModuleNotFoundError'>
Thu Sep 12 15:38:06 2019 - INFO | Exception value: No module named 'items'
Thu Sep 12 15:38:06 2019 - INFO | Exception Info:
  File "/usr/local/lib/python3.6/dist-packages/cobbler/module_loader.py", line 82, in __import_module
    blip = __import__("cobbler.modules.%s" % modname, globals(), locals(), [modname])
````

**Comment by watologo1** [comment]
I have a patch for this submitted already.

**Comment by watologo1** [comment]
Hmm, is it possible to explicitly mark this as  duplicate?
Here the reference:
Fix bad import #2150

**Comment by watologo1** [comment]
I merge the fix now, it's pretty important.
I will try to remember to send something for 3.0.1 as well.
I wonder how maintained/important 3.0.1 is. Is it used in any major distro?
